build_tools/packaging/promote_from_rc_to_final.py

```diff
@@ -0,0 +1,278 @@
+import argparse
```

Comment:
Add a shebang (https://en.wikipedia.org/wiki/Shebang_(Unix)) so the script is easier to run on Linux?
```python
#!/usr/bin/env python
```